.github/workflows/publish-swift.yml

```diff
@@ -0,0 +1,37 @@
+name: Publish JVM packages
+
+on:
+  push:
+    tags:
+      - "**"
+
+env:
+  CARGO_INCREMENTAL: 0
+  CARGO_NET_RETRY: 10
+  RUSTFLAGS: "-D warnings -W unreachable-pub"
+  RUSTUP_MAX_RETRIES: 10
+
+jobs:
+  publish-jvm:
+    if: github.repository == 'wireapp/core-crypto'
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: "Setup rust"
+        uses: dtolnay/rust-toolchain@stable
+        with:
+          targets: "aarch64-apple-ios,x86_64-apple-ios,aarch64-apple-ios-sim"
+      - name: Setup cargo-make
+        uses: davidB/rust-cargo-make@v1
+
+      - name: Build and Publish Swift package
+        run: |
+          cd crypto-ffi
+          cargo make create-swift-package
+          cd ../../
+          git clone https://github.com/wireapp/core-crypto-swift.git
+          cd core-crypto-swift
+          cp -r crypto-ffi/bindings/swift/* ./
+          git commit -am "chore: release of core-crypto $GITHUB_REF_NAME"
+          git tag $GITHUB_REF_NAME
+          git push https://${{ secrets.SWIFT_GITHUB_TOKEN }}@github.com/wireapp/core-crypto-swift.git
```

Comment:
No it can't. For SPM it has to be a git repo. The other option would be an xframework, but that can't be supported either because the SPM can't resolve the framework if we don't pack every possible arch, which we don't support at the moment.

Reply:
According to [github](https://github.blog/2019-06-03-github-package-registry-will-support-swift-packages/), they'll support swift packages, but that was announced in 2019 and nothing was released or announced since then, so I'm not sure they'll support it at any point in time.

Reply:
Gotcha